### PR TITLE
Enable pylint for travis

### PR DESCRIPTION
**Pull request by dwaynebailey**
- We use a Travis specific config
- Includes major sections which we pass and would like to keep passing
- Some minor fixes to get some checks passing

### CODE REVIEW

File: .pylint-travisrc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Travis specific pylint configuration #
+# ######################################
+#
+# For Travis we only care to fail if a test fails, so no fancy reporting
+#
+# Adding new enabled checks:
+# 1. They must pass
+# 2. They must pass reliably, no false positives
+# 3. Prefer a checker over and individual check
+# 4. We don't include anything that the devs can't agree on ito linting
```
Comment (phlax) [inline .pylint-travisrc]: well thats a defo gtm then!

Comment (unho) [inline .pylint-travisrc]: typo "ito"

File: .pylint-travisrc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+[MESSAGES CONTROL]
+disable=all
+# For checkers and indivodual checks see
```
Comment (unho) [inline .pylint-travisrc]: typo "indivodual"

**Comment by unho** [comment]
lgtm

**Pull request merged**